lib/stripe/subscriptions/invoice.ex

```diff
@@ -33,10 +33,12 @@ defmodule Stripe.Invoice do
           currency: String.t(),
           customer: Stripe.id() | Stripe.Customer.t(),
           date: Stripe.timestamp(),
+          default_source: String.t() | nil,
```

Comment:
Looks like one can pass this field to `update` and `create` as well

Reply:
👍